python/kwiver/vital/tests/test_activity_type.py

```diff
@@ -34,7 +34,6 @@
 
 """
 import unittest
-import nose.tools as nt
```

Comment:
Was this meant to be removed?

Reply:
Yes this was intentional, initially I removed it as I was going to refactor this test as the proof of concept, realized it implemented no nose functionality, and that import was therefor pointless so I left it out.